hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractKeysHTSSuite.java

```diff
@@ -115,6 +115,45 @@ public class ContractKeysHTSSuite extends HapiSuite {
+    private static final String STATIC_BURN_CALL_WITH_DELEGATE_CONTRACT_KEY_TXN =
+            "staticBurnCallWithDelegateContractKeyTxn";
+    private static final String NON_ZERO_TOKEN_BALANCE_DISSOCIATE_WITH_DELEGATE_CONTRACT_KEY_FAILED_TXN =
+            "nonZeroTokenBalanceDissociateWithDelegateContractKeyFailedTxn";
+    private static final String TOKEN_DISSOCIATE_WITH_DELEGATE_CONTRACT_KEY_HAPPY_TXN =
+            "tokenDissociateWithDelegateContractKeyHappyTxn";
+    private static final String CREATION_TX = "creationTx";
+    private static final String DISTRIBUTE_TX = "distributeTx";
+    private static final String TOKEN_ASSOCIATE = "tokenAssociate";
+    private static final String TOKEN_DISSOCIATE = "tokenDissociate";
+    private static final String BURN_CALL_AFTER_NESTED_MINT_CALL_WITH_PRECOMPILE_DELEGATE_CALL =
+            "burnCallAfterNestedMintCallWithPrecompileDelegateCall";
+    private static final String BURN_WITH_CONTRACT_KEY = "burn with contract key";
+    private static final String VANILLA_TOKEN_ASSOCIATE_TXN = "vanillaTokenAssociateTxn";
+    private static final String TOKEN = "Token";
+    private static final String DELEGATE_CONTRACT1 = "DelegateContract";
+    private static final String SERVICE_CONTRACT = "ServiceContract";
+    private static final String SECOND = "Second!";
+    private static final String DELEGATE_BURN_CALL_WITH_CONTRACT_KEY_TXN = "delegateBurnCallWithContractKeyTxn";
+    private static final String NESTED_ASSOCIATE_DISSOCIATE = "NestedAssociateDissociate";
+    private static final String STATIC_CONTRACT = "StaticContract";
+    private static final String BURN_CALL_AFTER_NESTED_MINT_CALL_WITH_PRECOMPILE_CALL =
+            "burnCallAfterNestedMintCallWithPrecompileCall";
+    private static final String BURN_DELEGATE_CALL_AFTER_NESTED_MINT_CALL_WITH_PRECOMPILE_DELEGATE_CALL =
+            "burnDelegateCallAfterNestedMintCallWithPrecompileDelegateCall";
+    private static final String BURN_DELEGATE_CALL_AFTER_NESTED_MINT_DELEGATE_CALL_WITH_PRECOMPILE_DELEGATE_CALL =
+            "burnDelegateCallAfterNestedMintDelegateCallWithPrecompileDelegateCall";
+    private static final String BURN_CALL_AFTER_NESTED_MINT_DELEGATE_CALL_WITH_PRECOMPILE_DELEGATE_CALL =
+            "burnCallAfterNestedMintDelegateCallWithPrecompileDelegateCall";
+    private static final String BURN_DELEGATE_CALL_AFTER_NESTED_MINT_CALL_WITH_PRECOMPILE_CALL =
+            "burnDelegateCallAfterNestedMintCallWithPrecompileCall";
+    private static final String BURN_DELEGATE_CALL_AFTER_NESTED_MINT_DELEGATE_CALL_WITH_PRECOMPILE_CALL =
+            "burnDelegateCallAfterNestedMintDelegateCallWithPrecompileCall";
+    private static final String BURN_CALL_AFTER_NESTED_MINT_DELEGATE_CALL_WITH_PRECOMPILE_CALL =
+            "burnCallAfterNestedMintDelegateCallWithPrecompileCall";
+    private static final String FIRST = "First!";
+    private static final String ANYBODY = "anybody";
```

Comment:
I changed the name. I feel like the methods are copy paste thats why. I don't see an issue to extract it especially that it removes the duplication and in case we would like diffrent string we can always change on specific method.